packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -182,8 +181,19 @@ type StoreConsistencyCheck<T> = {
   getSnapshot: () => T,
 };
 
+type EventFunctionState<T> = {
```

Comment:
Suggestions welcome on better naming for these types

Reply:
"Payload" maybe